lib/streamlit/web/cli.py

```diff
@@ -251,8 +254,8 @@ def _get_command_line_as_string() -> Optional[str]:
 
 def _main_run(
     file,
-    args: Optional[List[str]] = None,
-    flag_options: Optional[Dict[str, Any]] = None,
+    args: List[str] | None = None,
+    flag_options: Dict[str, Any] | None = None,
```

Comment:
nit: I think `List` can also be replaced with `list` with the annotations import

Reply:
Replaced 👍